test-utils/src/index.js

```diff
@@ -28,6 +28,21 @@ export function act(cb) {
 	if (flush) {
 		flush();
 	}
-	options.debounceRendering = Component.__test__previousDebounce;
 	options.requestAnimationFrame = previousRequestAnimationFrame;
 }
+
+/**
+ * Teardown test environment and reset preact's internal state
+ */
+export function teardown() {
+	if (Component.__test__drainQueue) {
```

Comment:
I believe this one is on me - do you think we could store this property elsewhere?  It could be changed to `options.__test__drainQueue`, or potentially just be a module-level variable.